Assignment 6

hhan-solvd opened the pull request:
I have renamed cardealership package to src, sorted classes by packages, added two more try-catch exceptions in CustomerEvaluation.java, added Manager.java to add Employee to EmployeeList, and added collections.

hhan-solvd committed:
Delete carDealership package

hhan-solvd committed:
Created collections

hhan-solvd committed:
Created collections

hhan-solvd committed:
Organized Exceptions in One Package

hhan-solvd committed:
Organized Exceptions in One Package

hhan-solvd committed:
Organized Interfaces in One Package

hhan-solvd committed:
Organized Interfaces in One Package

hhan-solvd committed:
Organized People-related Classes in One Package

hhan-solvd committed:
Organized People-related Classes in One Package

Also created Manager to add Employee to EmployeeList

hhan-solvd committed:
Organized Services-related Classes in One Package 

Organized services-related classes in one package and added collections

hhan-solvd committed:
Organized Services-related Classes in One Package

Organized services-related classes in one package and added collections

hhan-solvd committed:
Organized Vehicle-related Classes in One Package

Organized vehicle-related classes in one package and added collections

hhan-solvd committed:
Organized Vehicle-related Classes in One Package 

Organized vehicle-related classes in one package and added collections

hhan-solvd committed:
Added Collections

ankochergin commented on src/collections/CustomerList.java:
```diff
@@ -0,0 +1,22 @@
+
+import java.util.ArrayList;
+
+public class CustomerList {
```
I didn't get why you've created this, EmployeeList and TransactionList classes, I think we don't need them

hhan-solvd committed:
Deleted CustomerList.java

hhan-solvd committed:
Deleted EmployeeList.java

hhan-solvd committed:
Deleted EvaluationMap.java

hhan-solvd committed:
Deleted TransactionList.java

hhan-solvd committed:
Updated Customer.java

Added purchasedVehicle collection and setPurchasedVehicles().

hhan-solvd committed:
Updated Employee.java

Updated addCustomerToList() and addTransactionToList(), added addVehicleToCustomerPurchaseHistory(), and deleted unused import statements.

hhan-solvd committed:
Updated Manager.java

Updated addEmployeeToList(), added printAList() and printAMap() to print collections accordingly, and deleted unused import statements.

hhan-solvd committed:
Updated CustomerEvaluation.java

Updated addEvaluationToMap()

hhan-solvd committed:
Updated Dealership.java

Added collections and updated methods accordingly.

ankochergin commented on src/people/Manager.java:
```diff
@@ -0,0 +1,42 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Manager {
```
Manager is a type of Employee so I think it should extend Employee, what do you think?

ankochergin commented on src/people/Manager.java:
```diff
@@ -0,0 +1,42 @@
+        employeeList.add(employee.toString());
+    }
+
+    public void printAList(ArrayList<String> arrayList) {
+        for (String object : arrayList) {
+            LOGGER.info(object);
+        }
+    }
+
+    public void printAMap(HashMap<Integer, ArrayList<String>> hashMap) {
+        LOGGER.info(hashMap);
+    }
```
not sure that these methods relate to Manager entity
